x-pack/legacy/plugins/infra/public/pages/logs/analysis/page_results_content.tsx

```diff
@@ -180,7 +188,33 @@ export const AnalysisResultsContent = ({ sourceId }: { sourceId: string }) => {
             <EuiPageBody>
               <EuiPageContent>
                 <EuiPageContentBody>
-                  <LogRateResults isLoading={isLoading} results={logEntryRate} />
+                  {isFirstUse ? (
+                    <>
+                      <EuiCallOut
```

Comment:
How about we split this out into a separate component as a first step to shrinking this component down to a more easily readable size?